BUG: Made download_lyrics.py properly save into text files

iammax opened the pull request.

iammax committed:
BUG: Made download_lyrics.py properly save into text files

FORMATTING

FORMATTING: Improved readability of download_lyrics.py

EFulmer commented:
So here's the errors Travis is showing (you can see them here https://travis-ci.com/EFulmer/AIron-Maiden/builds/108571499 let me know if you can't):

```
flake8.main.application   MainProcess    663 INFO     Found a total of 2 violations and reported 2
1     E501 line too long (110 > 79 characters)
1     E722 do not use bare 'except'
2
```

(left hand column is number of times that specific style thing failed, so it's a total of 2).

`make format` will format your code, that kills item 1, `E501 line too long`.

`E722 do not use bare 'except'` is the tool complaining about  best practices. Usually you want to use the `except Exception as e:` exception handling syntax, for 2 reasons:

1. You can define different blocks for different exceptions, like a network error vs. a permissions error to read/write a specific directory
2. It lets you reference the exception in the `except` block, which is useful to say, log it. 

Official language reference here: https://docs.python.org/3.6/reference/compound_stmts.html#except 

Also, I'm real sorry about taking so long to follow up on this! :frowning_face:

iammax committed:
Formatting: Broke up excessively long line.
Best practices: Removed bare "except" statement and assigned it to a variable in order to more effectively catch errors

(Squish)

(Squash this)

iammax commented:
Pushed some fixes to address the errors thrown by travis.

Note: The excessively long line was mostly comment; make format did not see a problem with it. I had to do it manually. Perhaps the makefile needs something to get Black to format comments as well?
